lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -44,7 +46,10 @@
                 },
 
                 render: function () {
-                    this.$el.html(_.template(teamsTemplate));
+                    HtmlUtils.setHtml(
+                        this.$el,
+                        HtmlUtils.template(teamsTemplate)()
```

Comment:
This is another piece of magic from JQuery. A template is a function, and if you call ``this.$el.html`` with a function then it will invoke it for you with no arguments, and then set the HTML to the result. This works smoothly with templates that don't need any options.

I considered doing the same thing with ``HtmlUtils.template`` but I don't like implementing different behaviors based upon the argument's type. It seemed clearer to me to actually invoke the template with no parameters. I did modify ``HtmlUtils.template`` so that if options aren't provided that it acts the same as if you'd passed an empty object.